web/modules/custom/demo_users/demo_users.module

```diff
@@ -33,7 +33,7 @@ function demo_users_rdf_entity_insert(EntityInterface $entity) {
 
   // Check if there is already a membership. If one of the facilitators create
   // an entity, then the membership will already exist.
-  if (!Og::getMembership($entity, $facilitator)) {
+  if (!Og::getMembership($entity, $facilitator) && ($facilitator->id() != \Drupal::currentUser()->id())) {
```

Comment:
Actually, not the og membership. We are handling the relationship creation manually for rdf_entities in hooks (e.g. collection_entity_insert) since our rdf_entities do not have owners to have a membership created automatically.
Thus, when the collection facilitator demo user creates a collection, this membership creation is executed 2 times resulting in an exception.
Since this is a demo module, all exceptions should be handled here as the other hooks are the proper way to function.